pages/getting-started/developers.md

```diff
@@ -142,13 +142,15 @@ You can also email us directly at [uswebdesignstandards@gsa.gov](mailto:uswebdes
 
 ## JS customization
 
-The JavaScript for the standards is separated into components in the same manner as the visual interface which is all initialized with event handlers when the DOM is ready. These components are accessible as CommonJS modules that can be required in other JavaScript files which then must be fuilt for the browser. The components are currently not accessible in the global browser scope, but can be easily extended to be included by requiring `components` and setting it to a global scope:
+**Unfortunately, customizing the JavaScript for the standards currently requires NodeJS and a module bundler like Browserify or Webpack. We apologize for this inconvenience, and are working to resolve it in a future release of the Standards.**
+
+The JavaScript for the standards is separated into components in the same manner as the visual interface which is all initialized with event handlers when the DOM is ready. These components are accessible as CommonJS modules that can be required in other JavaScript files which then must be built for the browser. The components are currently not accessible in the global browser scope, but can be extended to be included by requiring `components` and setting it to a global scope:
 
 ```js
 window.uswds = require('./components');
 ```
 
-Each component has a standardized interface that can be used to extend it further. The primary methods each component has are as follows:
+Each component has a standardized interface that can be used to extend it further. The components store a HTML classname that's used to link HTML elements with the JS component, so when a component is initialized, it will search through the current HTML DOM finding all elements that match it's class and inialize the component JavaScript for those elements. The primary methods each component has are as follows:
```

Comment:
Minor typo, `it's -> its`.  Also, should `classname` be `class name`?  It might also be useful to just provide an example in parentheses, like `The components store a HTML class name (e.g. usa-accordion) ...` just to make things a bit more concrete.